sonic-pcied/scripts/pcied

```diff
@@ -157,6 +160,32 @@ class DaemonPcied(daemon_base.DaemonBase):
         if self.resultInfo is None:
             return
 
+        try:
+            stream = open(PLATFORM_PCIE_YAML_FILE, 'r')
+            dictionary = yaml.safe_load(stream)
+            for elem in dictionary:
+                bus = int(elem.get('bus'), 16)
+                device = int(elem.get('dev'), 16)
+                func = int(elem.get('fn'), 16)
+                pcieDeviceID = elem.get('id')
+
+                cmd = ("setpci -s %02x:%02x.%d 2.w" % (bus, device, func))
+                output, errs = subprocess.Popen(cmd, shell=True, executable='/bin/bash', stdout=subprocess.PIPE).communicate()
```

Comment:
Can you change this so that a shell isn't required? This would mean `cmd` looks something like `["setpci", "-s", "%02x:%02x.%d", "2.w"]` (before formatting arguments).

Reply:
This also means you may need to add in additional handling if the `setpci` binary doesn't exist or fails to be executed for whatever reason.

Reply:
Modified per our conversation in latest commit